ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceTestBase.java

```diff
@@ -68,29 +68,37 @@ public class JsonRpcHttpServiceTestBase {
   @ClassRule public static final TemporaryFolder folder = new TemporaryFolder();
   protected final JsonRpcTestHelper testHelper = new JsonRpcTestHelper();
 
-  private static final Vertx vertx = Vertx.vertx();
```

Comment:
The static variables caused test failures, because the tests are running in parallel and they were interfering with each other. The change to non-static variables resolves this issue.